### PR TITLE
Add a specialization for Cholesky adjoint on the GPU 

### PR DESCRIPTION
**Pull request by Red-Portal**
This addresses #1210

### CODE REVIEW

**Comment by Red-Portal** [comment]
News: Before merging this PR, some additional commits might be needed soon. See [this issue](https://github.com/JuliaDiff/ChainRules.jl/issues/629).

File: src/lib/array.jl [hunk 1]
```diff
@@ -593,6 +593,23 @@ end
   end
 end
 
+@init @require CUDA="052768ef-5323-5732-b1bb-66c8b64840ba" begin
+  Zygote.@adjoint function cholesky(Σ::Union{CUDA.CuMatrix, Symmetric{<:Real, <:CUDA.CuMatrix}}; check = true)
```
[the remark this answers is not in the document]
Comment (Red-Portal) [reply src/lib/array.jl:597]: Sure, I'll do it once the `ChainRules.jl` business is sorted out below.

**Comment by devmotion** [comment]
It seems this can be closed in favour of https://github.com/FluxML/Zygote.jl/pull/1114 now that the GPU issues and problems with that PR were fixed in CR 1.35.3.

**Comment by Red-Portal** [comment]
Indeed!

**Pull request closed without merging**